### ISSUE TITLE
Python ImportError

### ISSUE THREAD

**Issue by Ariela1127** [lucidrains/stylegan2-pytorch#196]
Hi,

I'm trying to execute stylegan2-pytorch on a Windows 10 machine, but when I run the "stylegan2_pytorch --data [path]" command, I get the following error:

`ImportError: cannot import name 'gcd' from 'fractions'`

Could anyone advise on how I can fix this error? I am using Python 3.9.1.

**Comment by lucidrains** [comment]
Hi Ariela! Do you want to give 1.5.7 a try?

**Comment by Ariela1127** [comment]
Hi @lucidrains - thanks for getting back to me :) 

As you suggested, I updated the version of stylegan2_pytorch to 1.5.7, but now I'm getting the following error:

`RuntimeError: The current Numpy installation fails to pass a sanity check due to a bug in the windows runtime. See this issue for more information: https://tinyurl.com/y3dm3h86`.

I also tried upgrading the package to 1.5.9, but then it was complaining about CUDA not being installed on my PC even though it definitely is - I followed the CUDA installation guide [here ](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html) to confirm that the installation was successfuly.

Any other ideas on what I could try?
